src/main/java/page_objects/BasePage.java

```diff
@@ -1,14 +1,30 @@
 package page_objects;
 
 import common.helpers.BrowserHelper;
-import common.helpers.ElementHelper;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 
 public class BasePage {
 
     //Locators
+    private final By tabLogin = By.xpath("//span[text()='Login']");
+    private final By lblWelcomeMessage = By.xpath("//div[@class='account']");
```

Comment:
can we use By.className?

Reply:
Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.